docs/groovy-script/getting_started/editors.md

```diff
@@ -42,6 +44,28 @@ This extension adds syntax validation, auto-completion for all installed mods, a
 6. Done
 
 
+## IntelliJ IDEA
+
+[IntelliJ IDEA](https://www.jetbrains.com/idea/) is an Integrated Development Environment created and distributed by JetBrains.
+The plugin will work with either Community Edition or Ultimate.
+
+[`Integer Limit`](https://github.com/IntegerLimit) has created a plugin, called [`GroovyScript`](https://plugins.jetbrains.com/plugin/25915-groovyscript),
+distributed by `IntegerLimit`.
+This plugin adds syntax validation, auto-completion for all installed mods, and hover information.
+
+The source code can be viewed [here](https://github.com/IntegerLimit/GroovyScriptPlugin).
+
+
+### Installation
+
+1. Open IntelliJ IDEA and install the [LSP4IJ](https://plugins.jetbrains.com/plugin/23257-lsp4ij) plugin and the [`GroovyScript`](https://plugins.jetbrains.com/plugin/25915-groovyscript) plugin
+2. Open the instance folder (typically `minecraft`) or the `groovy` folder of a modpack in IntelliJ IDEA.
+3. [Start the Language Server](#start-the-language-server) via the instructions above
+4. The port in the `groovyscript.cfg` config file must match the port setting in the IntelliJ IDEA plugin.
```

Comment:
Perhaps also mention how to support GrS versions pre 1.2.0, see my README for this.

Managing the server status is a bit annoying currently, but the docs doesn't have to mention this as I plan on fixing it eventually. But if you do want to mention this, it would be using the language servers tab (added by LSP4IJ), and then right clicking on the GroovyScript LSP entry to restart it. (in the case the server stops due to being unable to connect, or the instance being closed)

Reply:
fixed the incorrect info in step 5, but i went with a line earlier saying "hey more info on their readme" instead. there shouldnt be much that falls into this category?